homepage/models.py

```diff
@@ -53,6 +54,34 @@ class AppUser(models.Model):
     def __str__(self):
         return self.user.username
 
+    def toggle_user_activation(self):
+        self.user.is_active = not(self.user.is_active)
+
+    @staticmethod
+    # Creates a new appUser (without superuser permmisions) and saves it in the DB
+    def create_appUser(username, email, password):
+        appUser = AppUser()
+        user = User.objects.create_user(username, email, password)
+        appUser.user = user
+

```

Comment:
not sure what you mean Vera
we can use all of the fields from the built-in Django user class

is_active is automatically set to true because this is the Django beahviour